lib/launcher.go

```diff
@@ -4,18 +4,19 @@ import "os"
 import "os/exec"
 
 func main() {
-  launcherArgs := append([]string{os.Getenv("SPECTRON_LAUNCHER_PATH")}, os.Args[1:]...)
-  launcherCommand := exec.Command(os.Getenv("SPECTRON_NODE_PATH"), launcherArgs...)
-  launcherCommand.Stdout = os.Stdout
-  launcherCommand.Stderr = os.Stderr
+	launcherArgs := append([]string{os.Getenv("SPECTRON_LAUNCHER_PATH")}, os.Args[1:]...)
```

Comment:
Ahh... Vim automatically applies `gofmt` and I didn't notice it... I'll fix this.